addons/mail/static/src/models/thread/thread.js

```diff
@@ -1465,6 +1493,11 @@ function factory(dependencies) {
         serverFoldState: attr({
             default: 'closed',
         }),
+        /**
+         * This field containt all the suggested recipient that will be display
+         * inside the list.
```

Comment:
_"display inside the list"_ is off. This describes UI, which should usually be avoided in models (this is components concern). This is just the list of suggested recipients of this thread (the "info" is an unfortunate technical aspect, which should not be too apparent in comment, at least not in the _"what"_).